.docker/README.md

```diff
@@ -58,11 +58,9 @@ These containers are deployed to the Docker registry, so anyone can just
 `docker run -it leanprovercommunity/mathlib` to get a local lean+mathlib environment.
 
 There is a local script in `scripts/docker_push.sh` for deployment,
-but I have also set up `hub.docker.com` to watch the `docker` branch for updates
+but `hub.docker.com` is set up to watch the `master` branch for updates
 and automatically rebuild.
```

Comment:
@semorrison, should we just remove the last two lines of this? Are you the only one with credentials to dockerhub?

Reply:
Yes, let's remove. For now yes, I'm the only one with those credentials, but I will work on switching it to a shared credential that other maintainers can use.

Reply:
Updated